prov/tcp/src/tcpx_cq.c

```diff
@@ -83,20 +130,36 @@ int tcpx_cq_open(struct fid_domain *domain, struct fi_cq_attr *attr,
 		 struct fid_cq **cq_fid, void *context)
 {
 	int ret;
-	struct util_cq *cq;
+	struct tcpx_cq *tcpx_cq;
 
-	cq = calloc(1, sizeof(*cq));
-	if (!cq)
+	tcpx_cq = calloc(1, sizeof(*tcpx_cq));
+	if (!tcpx_cq)
 		return -FI_ENOMEM;
 
-	ret = ofi_cq_init(&tcpx_prov, domain, attr, cq,
+	if (!attr->size)
+		attr->size = TCPX_DEF_CQ_SIZE;
+
+	ofi_atomic_initialize64(&tcpx_cq->cq_free_size,
+				attr->size);
```

Comment:
I don't see that this value is used anywhere in this patch.

Reply:
I will remove in this patch and put it in apt patch.